### PR TITLE
Updated booleans for dragen 3.9 pipelines

### PR DESCRIPTION
**Pull request by alexiswl**
### dragen-alignment v3.9.3 tool/pipeline

* enable_map_align parameter renamed to enable_map_align_output
	*  --enable-map-align is set to true by default. 
	* --enable-map-align is set to true by default when --enable-map-align-output is set so no changes to outputs, just makes it more consistent with other workflows
	* enable_map_align_output parameter now mandatory
* enable_sort parameter now mandatory
* enable_duplicate_marking parameter now mandatory

### dragen germline v3.9.3 tool/pipeline

* enable_map_align_output parameter now mandatory
* enable_duplicate_marking parameter now mandatory

### dragen transcriptome v3.9.3 tool/pipeline

* enable_map_align_output parameter now mandatory
* enable_duplicate_marking parameter now mandatory

### dragen somatic v3.9.3 tool/pipeline

* enable_map_align_output parameter now mandatory
* enable_duplicate_marking parameter now mandatory

### dragen-wgs-qc pipeline v3.9.3 tool/pipeline

* enable_map_align_output parameter now mandatory
* enable_duplicate_marking parameter now mandatory
* enable_sort parameter now mandatory
* removed somalier step and outputs

### CODE REVIEW

File: tools/dragen-alignment/3.9.3/dragen-alignment__3.9.3.cwl [hunk 1]
```diff
@@ -361,13 +361,13 @@ inputs:
     type: int?
     inputBinding:
       prefix: "--Mapper.edit-seed-num"
-  enable_map_align:
-    label: enable map align
```
Comment (skanwal) [inline tools/dragen-alignment/3.9.3/dragen-alignment__3.9.3.cwl:365]: Although --enable-map-align is set to true by default but for clarity, I'd suggest we keep both inputs, `enable_map_align` and `enable_map_align_output`.

Comment (skanwal) [reply tools/dragen-alignment/3.9.3/dragen-alignment__3.9.3.cwl:365]: Same comment for other pipelines as well. I have checked Dragen docx, and it seems enable_map_align and enable_map_align_output are being explicitly called on the command-line. I am keen to follow the same and not rely on what are the default values for these important parameters.

**Review by skanwal — CHANGES_REQUESTED** [review]

File: tools/dragen-alignment/3.9.3/dragen-alignment__3.9.3.cwl (hunk 1 above)
Comment (alexiswl) [reply tools/dragen-alignment/3.9.3/dragen-alignment__3.9.3.cwl:365]: > I'd suggest we keep both inputs, enable_map_align and enable_map_align_output.

Not sure I follow. Running `--enable-map-align-output` forces `--enable-map-align` to be turned on.  

But the docs for enable-map-align suggest it is not relevant to us. 

Under MapperOptions, turning on `--enable-map-align` _enables use of BAM input files for mapper/aligner_

Comment (alexiswl) [reply tools/dragen-alignment/3.9.3/dragen-alignment__3.9.3.cwl:365]: Where as `--enable-map-align-output` is 'do you want a bam as output?'

Comment (skanwal) [reply tools/dragen-alignment/3.9.3/dragen-alignment__3.9.3.cwl:365]: Okay, so just enforcing the check here for generation of BAM i.e. should it be generated or not. 
Then makes sense. The initial comment in PR description isn't clear enough

```
--enable-map-align is set to true by default when --enable-map-align is set so no changes to outputs, just makes it more consistent with other workflows
```

**Review by skanwal — APPROVED** [review]
LGTM!

**Comment by alexiswl** [comment]
Failure occurred because requirements are fixed in https://github.com/umccr/cwl-ica/pull/152

**Pull request merged**